src/EPR.Calculator.API.Data/obj/Debug/net8.0/.NETCoreApp,Version=v8.0.AssemblyAttributes.cs

```diff
@@ -0,0 +1,4 @@
+// <autogenerated />
+using System;
```

Comment:
The obj directory can be added to .gitignore.

Reply:
Done